massa-protocol-worker-2/src/handlers/peer_handler/mod.rs

```diff
@@ -97,7 +119,7 @@ impl PeerManagementHandler {
                             }
                            }
                         },
-                        recv(receiver) -> msg => {
+                        recv(receiver_msg) -> msg => {
                             let (peer_id, message_id, message) = match msg {
                                 Ok((peer_id, message_id, message)) => (peer_id, message_id, message),
```

Comment:
Here you should deserialize the message (you can take example in other handlers) and manage it

Reply:
it's done on line 139 ?!

Reply:
Ah yes ok I'm dumb